packages/discovery-react-components/src/utils/document/transformEnrichment.ts

```diff
@@ -74,9 +74,7 @@ function createAttributeObjects(
         (att: Attributes): Attributes => createAttributeObject(att, attr)
       );
     } else {
-      return (
-        isValidAttribute(parsedObject[attr]) && createAttributeObject(parsedObject[attr], attr)
-      );
+      return hasLocationData(parsedObject[attr]) && createAttributeObject(parsedObject[attr], attr);
```

Comment:
This is not easy to read. If I understand this correctly, then just rewrite like this:

```javascript
} else if (hasLocationData(parsedObject[attr])) {
   createAttributeObject(parsedObject[attr], attr);
}
```

Also, `parsedObject[attr]` is used 4 times. Refactor into a local variable.